src/cartographer/config/overlays/sa-arn-annotation-overlay.yaml

```
@@ -0,0 +1,24 @@
+# Copyright 2021 VMware
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@ if/end hasattr(data.values, "arn_role_id"):
+#@overlay/match by=overlay.subset({"apiVersion":"v1","kind":"ServiceAccount","metadata":{"namespace":"cartographer-system", "name":"cartographer-conventions-controller-manager"}}), expects="1+"
```

there can only be one

```suggestion
#@overlay/match by=overlay.subset({"apiVersion":"v1","kind":"ServiceAccount","metadata":{"namespace":"cartographer-system", "name":"cartographer-conventions-controller-manager"}}), expects="1"
```